### PR TITLE
Added test_get_sites_by_box and performed some cleanup in wof model

### PR DESCRIPTION
**Pull request by lsetiawan**

### CODE REVIEW

File: wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py [hunk 1]
```diff
@@ -1,57 +1,52 @@
...
 from wof.dao import BaseDao
 import wof.examples.flask.odm2.timeseries.sqlalch_odm2_models as model
-from odm2api.ODM2 import models as odm2_models
-from sqlalchemy.orm import aliased
-from sqlalchemy import or_
-from sqlalchemy import and_
-from sqlalchemy.sql import func
 
-from sqlalchemy.orm import with_polymorphic
+from odm2api.ODM2 import models as odm2_models
```
Comment (emiliom) [inline wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py]: Are these two statements actually referring to different things?
```python
# line 9
import wof.examples.flask.odm2.timeseries.sqlalch_odm2_models as model
# vs
from odm2api.ODM2 import models as odm2_models
```

If they're the same, how about "merging" them?

File: wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py [hunk 2]
```diff
@@ -1,57 +1,52 @@
...
 
     def __del__(self):
+
         self.db_session.close()
 
     def get_all_sites(self):
-        s_rArr = self.db_session.query(odm2_models.Sites,odm2_models.TimeSeriesResults).\
+
+        s_rArr = self.db_session.query(odm2_models.Sites,
+                                       odm2_models.TimeSeriesResults).\
```
Comment (emiliom) [inline wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py]: FYI, at some point "in the future" I'd like to revisit why `odm2_models.TimeSeriesResults` is needed at all in the `query` method (as opposed to the `filter` method, where it's clearly needed), as we did when we were testing this stuff. I still suspect it's slowing down the query while not doing anything useful.

But I'm sure I'll forget or not prioritize it. If it ain't broken ...

File: wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py [hunk 3]
```diff
@@ -152,9 +161,10 @@ def __init__(self,aff_obj):
         self.Phone = aff_obj.PrimaryPhone
         self.Email = aff_obj.PrimaryEmail
         self.Address = aff_obj.PrimaryAddress
-        #self.City = 'San Diego'
-        #self.State = 'CA'
-        #self.ZipCode = '92122'
+        # self.City = 'San Diego'
+        # self.State = 'CA'
+        # self.ZipCode = '92122'
+
```
Comment (emiliom) [inline wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py]: Leave the commented out City, State and ZipCode, just as a reminder that they're probably missing and maybe should be added later (or maybe they're already in Address??)

File: wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py [hunk 4]
```diff
@@ -27,20 +29,21 @@ def __init__(self, v=None,
...
-    #TimeUnitsID = Column(Integer, ForeignKey('Units.UnitsID'))
-    #DataType = Column(String)
-    #GeneralCategory = wof_base.GeneralCategoryTypes.WATER_QUALITY
+    # SampleMedium = wof_base.SampleMediumTypes.NOT_RELEVANT
+    # ValueType = Column(String)
+    # IsRegular = Column(Boolean)
+    # TimeSupport = Column(Float)
+    # TimeUnitsID = Column(Integer, ForeignKey('Units.UnitsID'))
+    # DataType = Column(String)
+    # GeneralCategory = wof_base.GeneralCategoryTypes.WATER_QUALITY
```
Comment (emiliom) [inline wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py]: Leave in the commented out lines for ValueType, IsRegular and GeneralCategory. I'm not sure they're populated elsewhere. Having them here will serve as a reminder (hopefully) to look into it later. Not a priority at this time.

**Comment by emiliom** [comment]
I've reviewed it. See my comments / requests for small changes. Once you've implemented them (they're simple changes), feel free to merge the PR yourself. Thanks!

File: wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py (hunk 1 above)
Comment (lsetiawan) [inline wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py]: They are definitely not the same. One is ODM2 Models and the other is WOF Models

File: wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py (hunk 2 above)
Comment (lsetiawan) [inline wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py]: @emiliom So, don't query timeseriesresults at all?

File: wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py (hunk 4 above)
Comment (lsetiawan) [inline wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py]: Huh? They are commented out, I just put spaces to meet pep8 conventions.

File: wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py (hunk 3 above)
Comment (lsetiawan) [inline wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py]: Same thing as previous comment, not sure what you'd like me to do here, I just added some space.

File: wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py (hunk 1 above)
Comment (emiliom) [inline wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py]: Sorry, my bad! I get it now.

File: wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py (hunk 2 above)
Comment (emiliom) [inline wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py]: Ignore what I said. Not worth discussing or changing right now. 

But just FYI, this is what we tweaked and experimented with over a week ago in the notebook you lost.

File: wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py (hunk 3 above)
Comment (emiliom) [inline wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py]: Sigh. I must not be in a github review mode :(  For some reason I thought you were deleting the commented out lines. Ignore my comments, then.

File: wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py (hunk 2 above)
Comment (lsetiawan) [inline wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py]: Ah yes, I will maybe make an issue so that we remember to cover this in the future.

**Comment by emiliom** [comment]
I guess this means I didn't ask for any changes after all? If so, let me know, and I'll approve the review and merge the PR.

**Comment by lsetiawan** [comment]
@emiliom Indeed, No changes are necessary at this time. I will make an issue to cover the `TimeSeriesResults` stuff within the DAO.

**Comment by emiliom** [comment]
> I will make an issue to cover the TimeSeriesResults stuff within the DAO.

Thanks. I'll approve and merge now.

**Pull request merged**